### ISSUE TITLE
highlight_keywords and auto_filter and single filter

### ISSUE THREAD

**Issue by cjohnsonuk** [koalyptus/TableFilter#628]
If you have all of the above set then each individual letter of the search term you enter will be highlighted everywhere in the table.

How to reproduce:

On the page here :
`http://koalyptus.github.io/TableFilter/auto-filter.html#%7B%22col_0%22%3A%7B%22flt%22%3A%22bu%22%7D%7D`

open a console and type

```
tf.destroy();
var filtersConfig = {
        base_path: 'tablefilter/',
        auto_filter: {
            delay: 1100 //milliseconds
        },
        filters_row_index: 1,

highlight_keywords: true,
single_filter: true,

        state: true,
        alternate_rows: true,
        rows_counter: true,
        btn_reset: true,
        status_bar: true,
        msg_filter: 'Filtering...'
    };
    var tf = new TableFilter('demo', filtersConfig);
    tf.init();
```

this will enable all three.

Type in B then U and you will see that the Bs in BFA in the second column remain highlighted.  If you paste in BU then only the BUs are highlighted.  

When typing "term" into my own search box (private site) it highlights t's as follows
Crea _t_ ed By _T_ erm for Curren _t_  Academic Year
(ie only the first letter of the search term is highlighted)

When pasting "term" into my own search box (private site) it highlights as follows
Created By _Term_ for Current Academic Year

**Comment by koalyptus** [comment]
Tx for pointing this out @cjohnsonuk, I will have a look.

**Comment by koalyptus** [comment]
@cjohnsonuk this should now be fixed starting from v0.6.60

Cheers
